docs/understanding-radicle/how-it-works.md

```diff
@@ -83,92 +491,263 @@ carry a unique, shareable peer-to-peer identifier. The entirety of the project
-therefore replicating the data within their social graphs. In the context of a
-project, maintainers of a repository may choose to track the repositories of
-other owners (this is called a remote in Git terminology: a named reference to
-a remote repository). If the remote repository is found to track other remotes,
-the tracking repository shall also transitively track those, up to a
-configurable N degrees out.
-
-### Availability
+predetermined disjoint branch of the repository, by convention rad/id. This file
+contains important metadata such as the project name, list of maintainers, as
+well as any related links.
+
+The unit of replication is a repository, identified by a `PeerID` in the context
+of a project document (See [Data Model][dm]). The holder of the corresponding
+`DeviceKey` is referred to as the **maintainer** of the repository. Repositories
+belonging to the same project are represented locally as a single repository,
+identified by a Radicle URN (or [Radicle ID][ru] in the Upstream client). In the
+context of a project, the maintainer of a repository may choose to track the
+repositories of other peers (this is called a remote in git terminology: a named
+reference to a remote repository). If the remote repository is found to track
+other remotes, the tracking repository will also transitively track those, up to
+n degrees out.
+
+Therefore, a project on Radicle preserves the transitivity information of its
+remotes (i.e. via which tracked PeerID another PeerID is tracked).
+
+### Tracking
+Tracking is the backbone of collaboration as it drives the exchange of projects
+and their artifacts. In Radicle, peers track other peers and projects that they
+are interested in. This happens when a peer clones another peer's project or
+tracks a peer directly by adding them as a remote to their project via Upstream.
+
+Since peers represent seperate devices in the network, they each have their own
+view of the network. Each peer tracks this view of projects, identities, and
+data from connected peers in its own monorepo (See [Git Implementation][gi]).
+
+When a peer tracks another peer in the context of a project — say, if it clones
+another peer's project — it sets the intention to fetch and gossip the other
+peer's view of that project. This means all includes the project metadata, all
+working branches and commits, and changesets will be replicated and stored in
+the tracking peer's monorepo, so that it can be fetched and collaborated on.
+
+Specifically, this means the tracked peer will be added as a *remote* within the
+tracking peer's monorepo as a `$TRACKED_PEER_ID`, as seen in the following
+example:
+
+```rust
+# Owner of the monorepo
+let PEER_ID;
+
+# Peer tracked by $PEER_ID, either directly or transitively
+let TRACKED_PEER_ID;
+
+# Identity hash of the project or user
+let IDENTITY;
+
+# Identity hashes of certifiers of $IDENTITY
+let CERTIFIER[1..];
+
+$PEER_ID/refs/
+`-- namespaces
+    `-- $PROJECT_NAME
```

Comment:
```suggestion
    `-- $IDENTITY
```

alternatively, we use an example where the $ variables are replaced with legitimate values.